### ISSUE TITLE
Cheat Support? 

### ISSUE THREAD

**Issue by zakarupon** [Swordfish90/Lemuroid#119]
Curious if cheat support is something in the works for a future update. Thanks!

**Comment by Swordfish90** [comment]
Hi @zakarupon, I'm not personally a fan of cheats in games, but if anyone else wants to jump in and implement them for Lemuroid I will gladly accept the contribution upstream.

There is also [this](https://www.reddit.com/r/emulation/comments/j7gsog/yaba_sanshiro_saturn_emulator_suspended_from/) this matter that scares me a bit, so that's something that we need to handle properly in order to avoid issues with Google Play.

**Comment by zakarupon** [comment]
Thanks for the reply! I wasn't aware of the issue regarding cheats on the play store... Perhaps it's specifically for apps that have cheats built in to them, but having cheat support without any pre-installed would be fine since I've seen many Emulators on the app store like that. Either way, thanks for your amazing effort on such a great emulator and I hope someone with the right skills is willing to give it a shot if feasible.

**Comment by kranix0** [comment]
I've been using [FPseNG](https://play.google.com/store/apps/details?id=com.emulator.fpse64&hl=en&gl=US) from the Play Store and it has a full cheats system. So maybe the risk is overblown.

**Comment by kranix0** [comment]
If I may add a use case for this: I want to enjoy retro gaming with my young children, but they have motor skills issues and hence have problems with the controllers. Without cheats, games are unplayable for them. Cheats allow anybody to enjoy a game. See it as an accessibility feature.

**Comment by pointydev** [comment]
Cheat support doesn't just mean "cheats" in the traditional sense, they can also be used for fixing bugs and adding additional functionality, for example: https://the-sims-2-ds.fandom.com/wiki/Action_Replay_codes#Bugfix_Codes

If you're worried about Play store moderation, may I suggest hiding such functionality in a hidden menu such as tapping multiple times on a UI element? Alternatively, you could omit cheat management from Google Play builds altogether and rely on F-Droid/manual distribution. @Swordfish90

**Comment by kranix0** [comment]
Yes, "cheats" are actually mods that can also add useful features like better widescreen support.

RetroArch, Dolphin, Redream and FPseNG (mentioned above) have cheats support and have been on the Play Store for years, so I don't see how this could be a problem. 

It might be because they download the cheats (via the menus) rather than distributing in the package. This is the best way anyway.
